### PR TITLE
[207] Capture AKS system logs into a log analytics workspace

### PR DESCRIPTION
**Pull request by mrzta**
## What
We need to keep system logs to troubleshoot issues and incidents.

Log analytics is default on AKS. Diagnostic logs must be enabled and maybe container insights. We should implement it via terraform for platform-test, test and production clusters.

This is an initial implementation and may be refined later. The cost should be kept minimal.

## How to review
Check for the changes in the diagnostics settings are set in cluster 1
for example ..
Kubernetes API Server should have a check mark enabled

In the Logs please run this kql query
```
AzureDiagnostics
| where Category == "kube-apiserver"
```

### CODE REVIEW

**Comment by mrzta** [comment]
[Capture AKS system logs](https://trello.com/c/xEEJy1fI/207-capture-aks-system-logs)

File: custom_domains/documentation/aks-logs.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Retrieving Log Analytics Data with KQL for AKS Clusters
+
+Kusto Query Language (KQL) is a query language that you can use to retrieve data from Log Analytics workspaces for Azure Kubernetes Service (AKS) clusters. In this guide, we will explain how to write KQL queries to retrieve Log Analytics data and analyze it for your AKS cluster.
+
+## Prerequisites
```
Comment (saliceti) [inline custom_domains/documentation/aks-logs.md:5]: This section is not really needed as these elements will already be there

File: custom_domains/documentation/aks-logs.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Retrieving Log Analytics Data with KQL for AKS Clusters
+
+Kusto Query Language (KQL) is a query language that you can use to retrieve data from Log Analytics workspaces for Azure Kubernetes Service (AKS) clusters. In this guide, we will explain how to write KQL queries to retrieve Log Analytics data and analyze it for your AKS cluster.
+
```
Comment (saliceti) [inline custom_domains/documentation/aks-logs.md:4]: Maybe add a warning that it takes a few minutes for the logs to appear in tbhe workspace

File: custom_domains/documentation/aks-logs.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+- Retrieve all admin audit logs:
+
+    ```
+    AzureDiagnostics
+    | where Category == "kube-audit-admin"
+    ```
+
+These are just a few examples of the many queries that you can write using KQL to retrieve AKS Log Analytics data.
+
+## Conclusion
```
Comment (saliceti) [inline custom_domains/documentation/aks-logs.md:56]: This section sounds a bit marketing and is not really needed

**Review by saliceti — APPROVED** [review]

**Pull request merged**